examples/kubernetes/demo-regional-restricted-sc.yaml

```diff
@@ -6,6 +6,7 @@ provisioner: pd.csi.storage.gke.io
 parameters:
   type: pd-standard
   replication-type: regional-pd
+  labels: key1=value1,key2=value2
```

Comment:
Is it necessary to add labels to all the sample storageclasses if we already have demo-labeled-sc.yaml?

Given the amount these examples are copied/pasted I fear having them everywhere would lead to a lot of key1=value1 labels out in the world which disturbs me aesthetically :-P